wordpress-mobile/WordPress-iOS
# Fixes a crash due to an unregistered cell in Stats Insights

This PR adds `ViewsVisitorsRow` among the registered cells in `SiteStatsInsightsTableViewController`. Note that this registration would happen if the `statsNewInsights` feature flag was enabled, but it would seem that the cell is still used even if the same feature flag is disabled.

I observed the event specifically on iPad.

@sla8c , @Guarani, @frosty  not sure this was due to some wip in the stats revamp project, so it might require more work, but we should make sure the crash does not go into the next release.

Fixes #NA

To test:

- checkout/build/run another branch on iPad
- Play around with stats, switch between insights and days then insights again, switch to another item (e.g. Posts), then go back to Stats: you should observe a crash due to an unregistered `ViewsVisitorsLineChartCell`
- checkout/build/run this branch on iPad
- Repeat the test: the crash should not occur anymore

## Regression Notes
1. Potential unintended areas of impact
It should be fine, looks like we are registering a cell that was needed.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Since this might be an effect of an ongoing project, I asked review from the team that's working on that project

3. What automated tests I added (or what prevented me from doing so)
none

PR submission checklist:
